pkg/emscripten/libretro/index.html

```diff
@@ -81,11 +93,29 @@
                            <a class="dropdown-item" href="." data-core="stella">Stella</a>
                            <a class="dropdown-item" href="." data-core="tgbdual">TGB Dual</a>
                            <a class="dropdown-item" href="." data-core="theodore">Theodore (Thomson TO8/TO9)</a>
+                           <a class="dropdown-item" href="." data-core="tic80">TIC-80</a>
                            <a class="dropdown-item" href="." data-core="tyrquake">TyrQuake</a>
+                           <a class="dropdown-item" href="." data-core="uzem">UZEM</a>
+                           <a class="dropdown-item" href="." data-core="vaporspec">Vaporspec</a>
                            <a class="dropdown-item" href="." data-core="vba_next">VBA Next</a>
                            <a class="dropdown-item" href="." data-core="vecx">Vecx</a>
+                           <a class="dropdown-item" href="." data-core="vice_x64">VICE x64</a>
+                           <a class="dropdown-item" href="." data-core="vice_x64sc">VICE x64sc</a>
+                           <a class="dropdown-item" href="." data-core="vice_x128">VICE x128</a>
+                           <a class="dropdown-item" href="." data-core="vice_xcbm2">VICE xcbm2</a>
+                           <a class="dropdown-item" href="." data-core="vice_xcbm5x0">VICE xcbm5x0</a>
+                           <a class="dropdown-item" href="." data-core="vice_xpet">VICE xPET</a>
+                           <a class="dropdown-item" href="." data-core="vice_xplus4">VICE xPlus4</a>
+                           <a class="dropdown-item" href="." data-core="vice_xscpu64">VICE xscpu4</a>
+                           <a class="dropdown-item" href="." data-core="vice_xvic">VICE xVIC</a>
+                           <a class="dropdown-item" href="." data-core="vitaquake2">Vita Quake2</a>
+                           <a class="dropdown-item" href="." data-core="vitaquake2-roque">Vita Quake2 (rogue)</a>
+                           <a class="dropdown-item" href="." data-core="vitaquake2-xatrix">Vita Quake2 (xatrix)</a>
+                           <a class="dropdown-item" href="." data-core="vitaquake2-zaero">Vita Quake2 (zaero)</a>
                            <a class="dropdown-item" href="." data-core="virtualjaguar">Virtual Jaguar</a>
                            <a class="dropdown-item" href="." data-core="wasm4">WASM4</a>
+                           <a class="dropdown-item" href="." data-core="xmil">XMillenium</a>
```

Comment:
Is the core id used for XMillenium correct?

```suggestion
                           <a class="dropdown-item" href="." data-core="x1">XMillenium</a>
```